src/manifolds/ProbabilitySimplex.jl

```diff
@@ -0,0 +1,369 @@
+@doc raw"""
+    ProbabilitySimplex{n} <: AbstractEmbeddedManifold{ℝ,DefaultEmbeddingType}
+
+The (relative interior of) the probability simplex is the set
+````math
+Δ^n := \biggl\{ p ∈ ℝ^{n+1}\ \big|\ p_i > 0 \text{ for all } i=1,…,n+1,
+\text{ and } ⟨\mathbb{1},p⟩ = \sum_{i=1}^{n+1} p_i = 1\biggr\},
+````
+where $\mathbb{1}=(1,…,1)^{\mathrm{T}}∈ ℝ^{n+1}$ denotes the vector containing only ones.
+
+This set is also called the unit simplex or standard simplex.
+
+The tangent space is given by
+````math
+T_pΔ^n = \biggl\{ X ∈ ℝ^{n+1}\ \big|\ ⟨\mathbb{1},X⟩ = \sum_{i=1}^{n+1} X_i = 0 \biggr\}
+````
+
+We employ this set and its tangent spaces with the [`FisherRaoMetric`](@ref) exploiting
+the isometry to the $n$-sphere of radius 2, i.e. a scaled version of the [`Sphere`](@ref).
+The corresponding diffoemorphism $\varphi\colon \mathbb Δ^n \to \mathcal N$,
+where $\mathcal N \subset 2𝕊^n$ is given by %\varphi(p) = 2\sqrt{p}$.
+
+This implementation follows the notation in [^ÅströmPetraSchmitzerSchnörr2017]
+
+[^ÅströmPetraSchmitzerSchnörr2017]:
+    > F. Åström, S. Petra, B. Schmitzer, C. Schnörr: “Image Labeling by Assignment”,
+    > Journal of Mathematical Imaging and Vision, 58(2), pp. 221–238, 2017.
+    > doi: [10.1007/s10851-016-0702-4](https://doi.org/10.1007/s10851-016-0702-4)
+    > arxiv: [1603.05285](https://arxiv.org/abs/1603.05285).
+"""
+struct ProbabilitySimplex{n} <: AbstractEmbeddedManifold{ℝ,DefaultEmbeddingType} end
+
+ProbabilitySimplex(n::Int) = ProbabilitySimplex{n}()
+
+"""
+    SoftmaxRetraction <: AbstractRetractionMethod
+
+Describes a retraction that is based on the Softmax function.
```

Comment:
It's not usually capitalized
```suggestion
Describes a retraction that is based on the softmax function.
```